src/lib/soy-generator.ts

```diff
@@ -540,7 +542,34 @@ export class SoyConverter {
               continue;
             }
             const textLiterals = value.split(markerRegex);
-            commands.push(new ast.RawText(` ${name}="${textLiterals[0]}`));
+
+            const isBound = textLiterals.length > 1;
+            if (!isBound) {
+              commands.push(new ast.RawText(` ${name}="${value}"`));
+              continue;
+            }
+
+            if (isBoolean) {
+              // There should be one expression.
```

Comment:
This should be checked. Also, it's not immediately clear how the comment relates to the next line. Try:

```js
// Check that there's one expression
```
or
```js
// This assumes that there's one expression because...
```